Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -155,6 +154,16 @@
                     "order": 40000
                 },
 
+                // Configurable starting prioirity for all Gem scan folders to use.
+                // Each Gem scan folder added will increment this.
+                "GemScanFolderStartingPriorityOrder": 100,
```

Comment:
Can you update this comment to mention that due to incrementing this value, every successive gem has a lower priority to each other?
```suggestion
                // Configurable starting prioirity for all Gem scan folders to use.
                // Each Gem scan folder added will increment this.
                // NOTE: Each successive gem priority will be lower than the last
                "GemScanFolderStartingPriorityOrder": 100,
```

Reply:
Done.